projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/vocabulary/ControlledVocabulary.java

```diff
@@ -0,0 +1,39 @@
+package life.qbic.projectmanagement.domain.project.vocabulary;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * object for controlled vocabulary information.
+ *
+ * @since 1.0.0
+ */
+public class ControlledVocabulary {
+
+  public ControlledVocabulary(String identifier,
+      Map<String,String> vocabularyTermsByName) {
+    this.identifier = identifier;
+    this.vocabularyTermsByName = vocabularyTermsByName;
+  }
+
+  final private String identifier;
+
+  private Map<String,String> vocabularyTermsByName;
+
+  public String getIdentifier() {
+    return identifier;
+  }
+
+  public List<String> getVocabularyTermNames() {
+    return new ArrayList<>(vocabularyTermsByName.keySet());
```

Comment:
Question: I'd expect the map ordering to be switched with the database identifier as key and the name as value, since it's the value behind the identifiers that is of interest of us?

Reply:
That's fair. I did it this way because I re-use it to map it back to the code later. (method that is not used atm)